pkg/updater/local_api.go

```diff
@@ -12,18 +12,32 @@ import (
 	"net"
 	"net/http"
 	"os"
-	"path"
 
 	"github.com/gorilla/mux"
 
+	"github.com/DataDog/datadog-agent/pkg/updater/repository"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
-type apiResponse struct {
-	Error *apiError `json:"error,omitempty"`
+const (
+	defaultSocketPath = defaultRepositoriesPath + "/updater.sock"
+)
+
+// StatusResponse is the response to the status endpoint.
+type StatusResponse struct {
+	APIResponse
+	Version  string                      `json:"version"`
+	Packages map[string]repository.State `json:"packages"`
+}
+
+// APIResponse is the response to an API request.
+type APIResponse struct {
+	Error *APIError `json:"error,omitempty"`
 }
 
-type apiError struct {
+// APIError is an error response.
+type APIError struct {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Code, maybe? Or do we always send 500s for now?